relay/exchange/orderbook.py

```diff
@@ -83,6 +83,14 @@ def order_filled(self, orderhash: bytes, filled_maker_amount: int, filled_taker_
         if self._db is not None:
             return self._db.order_filled(orderhash, filled_maker_amount, filled_taker_amount)
 
+    def order_cancelled(self, orderhash: bytes, cancelled_maker_amount: int, cancelled_taker_amount: int):
```

Comment:
Please add return type and change it to not return the called function, because it always returns None. Same in order_filled, was badly written by me I guess.